example_case_folders/07_floris_standin_and_solar_pysam/hercules_controller_input_000.yaml

```diff
@@ -0,0 +1,63 @@
+# Input YAML for emy_python
+
+# Name
+name: example_000
+
+###
+# Describe this emulator setup
+description: Floris and Solar PV
+
+dt: 0.5
+
+hercules_comms:
+
+  amr_wind:
+
+    wind_farm_0:
+      type: amr_wind_local #options are amr_wind or amr_wind_local
+      amr_wind_input_file: amr_input.inp
+
+  helics:
+
+    config:
+        name: hercules # What is the purpose of this name
+        use_dash_frontend: False
+        KAFKA: False
+        KAFKA_topics: EMUV1py
+        helics: 
+            # deltat: 1 # This will be assigned in software
+            subscription_topics: [status]
+            publication_topics: [control]
+            endpoints: []
+            helicsport: 32000
+        publication_interval: 1
+        endpoint_interval: 1
+        starttime: 0
+        stoptime: 100 # must be at least 2*dt smaller than last timestep in weather file
+        Agent: ControlCenter
+
+py_sims:
+
+  solar_farm_0: # The name of py_sim object 1
+
+    py_sim_type: SolarPySAM
+    weather_file_name: NonAnnualSimulation-sample_data-interpolated-daytime.csv
+    system_info_file_name: 100MW_1axis_pvsamv1.json
+    lat: 39.7442
+    lon: -105.1778
+    elev: 1829
+
+    # capacity: 100 # MW
+
+    initial_conditions:
+
+      power: 25 # MW
+      dni: 1000
+
+controller:
+
+external_data_file: solar_power_reference.csv
```

Comment:
This file is missing from the repo, I think

Reply:
Added!